language/Portuguese_Brazil/strings.po

```diff
@@ -3274,3484 +3282,3523 @@ msgstr "[Nenhum]\n"
 #, c-format
 msgctxt "view-name"
 msgid "Screen %1$u Standard (%2$u:%3$u)"
-msgstr ""
+msgstr "Tela %1$u Padrão (%2$u:%3$u)"
 
 #: src/emu/render.cpp:1873
 #, c-format
 msgctxt "view-name"
 msgid "Screen %1$u Pixel Aspect (%2$u:%3$u)"
-msgstr ""
+msgstr "Tela %1$u Aspecto dos Pixeis (%2$u:%3$u)"
 
 #: src/emu/render.cpp:1895 src/emu/render.cpp:2030
 msgctxt "view-name"
 msgid "Cocktail"
-msgstr ""
+msgstr "Coquetel"
 
 #: src/emu/render.cpp:2015
 msgctxt "view-name"
 msgid "Left-to-Right"
-msgstr ""
+msgstr "Esquerda-pra-direita"
 
 #: src/emu/render.cpp:2016
 msgctxt "view-name"
 msgid "Left-to-Right (Gapless)"
-msgstr ""
+msgstr "Esquerda-pra-Direita (Sem Espaços)"
 
 #: src/emu/render.cpp:2017
 msgctxt "view-name"
 msgid "Top-to-Bottom"
-msgstr ""
+msgstr "Do Topo-até-o-Rodapé"
 
 #: src/emu/render.cpp:2018
 msgctxt "view-name"
 msgid "Top-to-Bottom (Gapless)"
-msgstr ""
+msgstr "Do Topo-até-o-Rodapé (Sem Espaço)"
 
 #: src/emu/render.cpp:2069
 #, c-format
 msgctxt "view-name"
 msgid "%1$u×%2$u Left-to-Right, Top-to-Bottom"
-msgstr ""
+msgstr "%1$u×%2$u Esquerda-pra-Direita, Do Topo-até-o-Rodapé"
 
 #: src/emu/render.cpp:2078
 #, c-format
 msgctxt "view-name"
 msgid "%1$u×%2$u Left-to-Right, Top-to-Bottom (Gapless)"
-msgstr ""
+msgstr "%1$u×%2$u Esquerda-pra-Direita, Do Topo-até-o-Rodapé (Sem Espaço)"
 
 #: src/emu/render.cpp:2087
 #, c-format
 msgctxt "view-name"
 msgid "%1$u×%2$u Top-to-Bottom, Left-to-Right"
-msgstr ""
+msgstr "%1$u×%2$u Do Topo-até-o-Rodapé, Esquerda-pra-Direita"
 
 #: src/emu/render.cpp:2096
 #, c-format
 msgctxt "view-name"
 msgid "%1$u×%2$u Top-to-Bottom, Left-to-Right (Gapless)"
-msgstr ""
+msgstr "%1$u×%2$u Do Topo-até-o-Rodapé, Esquerda-pra-Direita (Sem Espaço)"
 
 #: src/emu/inpttype.ipp:22
 msgctxt "input-name"
 msgid "P1 Up"
-msgstr ""
+msgstr "Jogador 1 Pra Cima"
 
 #: src/emu/inpttype.ipp:23
 msgctxt "input-name"
 msgid "P1 Down"
-msgstr ""
+msgstr "Jogador 1 Pra Baixo"
 
 #: src/emu/inpttype.ipp:24
 msgctxt "input-name"
 msgid "P1 Left"
-msgstr ""
+msgstr "Jogador 1 Esquerdo"
 
 #: src/emu/inpttype.ipp:25
 msgctxt "input-name"
 msgid "P1 Right"
-msgstr ""
+msgstr "Jogador 1 Direito"
 
 #: src/emu/inpttype.ipp:26
 msgctxt "input-name"
 msgid "P1 Right Stick/Up"
-msgstr ""
+msgstr "Jogador 1 Analógico Direito/Pra Cima"
 
 #: src/emu/inpttype.ipp:27
 msgctxt "input-name"
 msgid "P1 Right Stick/Down"
-msgstr ""
+msgstr "Jogador 1 Analógico Direito/Pra Baixo"
 
 #: src/emu/inpttype.ipp:28
 msgctxt "input-name"
 msgid "P1 Right Stick/Left"
-msgstr ""
+msgstr "Jogador 1 Analógico Direito/Esquerdo"
 
 #: src/emu/inpttype.ipp:29
 msgctxt "input-name"
 msgid "P1 Right Stick/Right"
-msgstr ""
+msgstr "Jogador 1 Analógico Direito/Direito"
```

Comment:
These inputs (and the corresponding right stick inputs, and the inputs for other players) are not for dual analog sticks – they are for dual digital sticks as used by games like Crazy Climber or Smash TV.

Reply:
I dont understand what you mean, those were translated correctly.

Reply:
I think, because this is for digital joysticks, that cuavas was surprised to see the word "analógico" ... but if it is correct, then it is correct. It seems "analógico" is being used here as noun, not adjective.

Reply:
Correct me if I’m wrong, but isn’t “analógico” Portuguese for the English “analog”?  As I understand, the Portuguese for “analog joystick” is “joystick analógico” and the Portuguese for “digital joystick” is “joystick digitais”.

MAME’s controls like “P1 Right Stick/Right” are for digital joysticks, not analog joysticks, so unless my understanding of “analógico” is completely wrong, the translation appears to be incorrect to me.

Reply:
stilett0/cuavas - thanks for your points, i made changes to improve these as other small changes to strings that i think it's better for the translation.